### PR TITLE
Some fixes: not needed varbinary and missing UPDATE::SUCCESS

### PR DESCRIPTION
**Pull request by Quix0r**
Some fixes:
- varbinary() is not needed when clear-text words like 'xmpp' are used for them,   it also hinders using external tools like Adminer/phpMyAdmin to search for   them as e.g. Adminer wraps a HEX() call (SQL) around `k` and `cat` (see table   `pconfig`)
- added missing UPDATE::SUCCESS

### CODE REVIEW

File: update.php [hunk 1]
```diff
@@ -747,4 +747,5 @@ function pre_update_1365()
 		return Update::FAILED;
 	}
 
+	return UPDATE::SUCCESS;
```
Comment (AlfredSK) [inline update.php]:
```suggestion
	return Update::SUCCESS;
```

**Comment by Quix0r** [comment]
Thank you for finding.

**Review by MrPetovan — APPROVED** [review]

**Comment by annando** [comment]
The values are `varbinary` because of the key length. Depending on the SQL server configuration this could create errors now and could lead to failed updates.

**Comment by MrPetovan** [comment]
Even with `50` and `100` for field lengths?

**Comment by annando** [comment]
Currently I'm unsure, possibly we should check.

**Comment by AlfredSK** [comment]
When I remember correctly the issue with varchar key length can happen with field length > 192.

Edit: Didn't we force the usage of InnoDB file format Barracuda lately? Because with InnoDB row format dynamic or compressed (mandatory when using Barracuda) the key length isn't an issue anymore.

**Comment by annando** [comment]
Yes, we now notify the admin about this issue. That's why I meant that it could be an issue for people who update from older versions.

**Pull request merged**